server/src/main/java/org/elasticsearch/action/support/SubscribableListener.java

```diff
@@ -35,6 +36,61 @@
  * <p>
  * Similar to {@link ListenableActionFuture} and {@link ListenableFuture} except for its handling of exceptions: if this listener is
  * completed exceptionally then the exception is passed to subscribed listeners without modification.
+ * <p>
+ * Often this will be used to chain together a sequence of async actions, similarly to {@link CompletionStage} (without the
+ * {@code catch (Throwable t)}), such as in the following example:
+ * <pre>
+ * private void exampleAsyncMethod(String request, List&lt;Long> items, ActionListener&lt;Boolean> finalListener) {
+ *     SubscribableListener
+ *
+ *         // Start the chain and run the first step by creating a SubscribableListener using newForked():
+ *         .&lt;String>newForked(l -> firstAsyncStep(request, l))
```

Comment:
I'm reading that using a combination of `<pre> + {@code}` can help make this large section of code render correctly the `<` without using the HTML escaping. Could we do that? It'd improve readability (without having to render the javadoc).

Reply:
TIL :) see b8eba94